templates/partials/modal_messagehistory.haml

```diff
@@ -7,7 +7,7 @@
     .message-history
       .action.stackitem{ ng-repeat:"action in actions" }
         .action-time
-          [[ action.created_on | autodate ]]
+          <cp-tooltip display-text="[[ action.created_on | autodate]]" tooltip-text="[[ action.created_on | fulldate ]]" position="top-right"><cp-tooltip>
```

Comment:
a) that was my original plan but I couldn't get the directive to work when I tried to do that. I was probably doing it wrong since I don't really understand this stuff
b) currently this directive could be used to add tooltips to any text, it isn't restricted to dates. Which of course also means that it's not enforcing that dates always be displayed this way but it's just facilitating it.